### ISSUE TITLE
Allow to disable ssh compression

### ISSUE THREAD

**Issue by s-m-e** [masc3d/btrfs-sxbackup#54]
Disabling ssh's own compression (which might be on by default in users' settings) speeds things up considerably in certain scenarios. You might want to add `-o Compression=no` [here](https://github.com/masc3d/btrfs-sxbackup/blob/83149847bcb46694a16031105e9de4010ff47445/btrfs_sxbackup/shell.py#L35).

**Comment by masc3d** [comment]
could optionally override this but only in case `lzop` compression is enabled via backup job.

alternatively this could also be configured via `ssh_config` ref #55

**Comment by s-m-e** [comment]
Yep, same as #55. Though, in this case, if you know your btrfs data is already compressed, you could just automatically deactivate ssh's compression in your code. It's one less thing to configure for your users and it makes a sure difference.
